File: packages/android/formbricksSDK/src/main/java/com/formbricks/formbrickssdk/model/javascript/EventType.kt
package com.formbricks.formbrickssdk.model.javascript

import com.google.gson.annotations.SerializedName

enum class EventType {
    @SerializedName("onClose")  ON_CLOSE,
    @SerializedName("onDisplayCreated") ON_DISPLAY_CREATED,
    @SerializedName("onResponseCreated") ON_RESPONSE_CREATED,
    @SerializedName("onFilePick") ON_FILE_PICK,
    @SerializedName("onSurveyLibraryLoadError") ON_SURVEY_LIBRARY_LOAD_ERROR
}

File: packages/ios/FormbricksSDK/FormbricksSDK/Model/Javascript/EventType.swift
enum EventType: String, Codable {
    case onClose = "onClose"
    case onDisplayCreated = "onDisplayCreated"
    case onResponseCreated = "onResponseCreated"
    case onOpenExternalURL = "onOpenExternalURL"
    case onSurveyLibraryLoadError = "onSurveyLibraryLoadError"
}
